terraform/meom.tfvars

```diff
@@ -0,0 +1,15 @@
+prefix     = "meom"
+project_id = "meom-ige-cnrs"
+
+# Inane CPU requests mean we need at least 3 CPUs for a base node?!?!
```

Comment:
Why?? Do you have more context about this requirement?

Reply:
I've updated the comment with a new instance size + some context!